pyhafas/profile/db/helper/request.py

```diff
@@ -0,0 +1,43 @@
+import json
+
+import requests
+
+from pyhafas.profile import ProfileInterface
+from pyhafas.profile.base.helper.request import BaseRequestHelper
+from pyhafas.profile.base.mappings.error_codes import BaseErrorCodesMapping
+from pyhafas.types.hafas_response import HafasResponse
+from tenacity import (
+    retry,
+    retry_if_exception_type,
+    wait_exponential,
+    stop_after_attempt,
+)
+
+
+class DBRequestHelper(BaseRequestHelper):
```

Comment:
I think there is no reason why to only include this for the DB hafas, so you might want to implement this directly in `BaseRequestHelper`.

Reply:
Ok. Do you think the `retry` and `backoff` values should be optional?

Reply:
Don't think so. Maybe some method of (de)activating retry would be nice (could be enabled by default), e.g. in the profile (as it's easy to customize them without forking pyhafas).